tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,187 @@
+.. _doc_openxr_passthrough:
+
+AR / Passthrough
+================
+
+Augmented Reality is supported through various methods depending on the capabilities of the hardware.
+
+Headsets such as the Magic Leap and glasses such as TiltFive show the rendered result on transparent displays
+allowing the user to see the real world.
+
+Headsets such as the Quest, HTC Elite, and Lynx R1 implement this through a technique called passthrough,
+where cameras record the real world and these images are used as the background on top of which our rendered
+result is used.
+
+.. note::
+
+    The way specifically passthrough is implemented has vast differences between different platforms.
+
+    In Godot 4.3 we have implemented a unified approach that is explained on this help page
+    so you don't need to worry about these differences, the :ref:`XRInterface <class_xrinterface>`
+    implementation is now responsible for applying the correct platform dependent method [#]_.
+
+    For headsets such as the Meta Quest and HTC Elite you will need to use the
+    `OpenXR vendors plugin v3.0.0 <https://github.com/GodotVR/godot_openxr_vendors/releases>`__
+    or later to enable passthrough.
+
+    For backwards compatibility the old API for passthrough is still available but it is recommended
+    to follow the new instructions below.
+
+Environment blend modes
+-----------------------
+
+The way we configure VR or AR functionality is through setting the environment blend mode.
+This mode determines how the (real world) environment is blended with the virtual world.
+
+.. list-table:: Blend modes
+  :widths: 35 65
+  :header-rows: 1
+
+  * - Blend mode
+    - Description
+  * - XR_ENV_BLEND_MODE_OPAQUE
+    - The rendered image is opaque, we do not see the real world. We're in VR mode.
+      This will turn off passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ADDITIVE
+    - The rendered image is added to the real world. This mode generally applies to AR only devices
+      such as the Hololens where the optics are not able to obscure the real world.
+      This will turn on passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ALPHA_BLEND
+    - The rendered image is alpha blended with the real world.
+      This is supported on devices that can fully obscure the real world
+      and generally the preferred way to do AR.
```

Comment:
The sentence is a bit confusing; the first section states `This is supported on devices that can fully obscure the real world` leading to the assumption it's only supported on video-passthrough devices.
But the second section asserts this is `generally the preferred way to do AR`, leading to the assumption it can be used on both types of devices.

Reply:
I think that is an assumption on your part, there are see-through devices that can fully obscure the real world.